backend/dal/Migrations/v01.06.01/Up/PostUp/01-BuildingPredominateUses.sql

```diff
@@ -0,0 +1,18 @@
+PRINT 'Add Marina to Building Predominate Uses'
+
+--Check in case of rollback/rollforward
+BEGIN
```

Comment:
Don't think you need to have the extra `BEGIN` and `END` wrapping the whole thing